### PR TITLE
Fix ScrollPanel LayoutContent

### PR DESCRIPTION
**Pull request by garryspins**

### CODE REVIEW

**Review by bludotbat — APPROVED** [review]
be sure to use proper git convention next time

**Review by bludotbat — APPROVED** [review]

**Comment by TomDotBat** [comment]
> be sure to use proper git convention next time

Just rebase before merging

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.2.0
\ No newline at end of file
+1.2.1
```
Comment (TomDotBat) [inline VERSION:1]: You forgot to bump the version in the library loader.
Next time don't perform these changes in a PR, I will handle versioning/releases.

**Comment by TomDotBat** [comment]
@bludotbat can we avoid merging PRs without my input unless it's urgent please.

**Pull request merged**